Automattic/vip-go-mu-plugins
# Add support for WordPress 5.5 environment type

## Description

Add support for [WordPress 5.5 environment type](https://make.wordpress.org/core/2020/07/24/new-wp_get_environment_type-function-in-wordpress-5-5/). We set the environment type to exactly the same value as the VIP Go environment name.

WordPress expects the environment name to be one of "production", "staging" or "development". When the VIP Go environment cannot be mapped to any of these standard types, it's added to the list of accepted environments for that site.

## Checklist


## Steps to Test

1. Check out PR and setup dev environment using WordPress 5.5
1. Get the result of `wp_get_environment_type()`
1. Verify that it matches the definition of `VIP_GO_APP_ENVIRONMENT`
